cli/unit-test.sh

```diff
@@ -7,3 +7,6 @@ FILE_PATH=$(
 
 cd "$FILE_PATH"/src/core/parse || exit
 go test .
+
+cd "$FILE_PATH"/src/util/slice || exit
+go test .
```

Comment:
what about making these verbose ? 
```suggestion
go test . -v

cd "$FILE_PATH"/src/util/slice || exit
go test . -v
```

Reply:
I don't like doing verbose mode with running tests. When a test fails, it will still give output (without verbose mode). There is no reason to see output from passing tests, we only ever want to see output from failed tests.